cmd/config/main.go

```diff
@@ -30,63 +26,95 @@ import (
 	"github.com/skip-mev/slinky/providers/websockets/kucoin"
 	"github.com/skip-mev/slinky/providers/websockets/mexc"
 	"github.com/skip-mev/slinky/providers/websockets/okx"
+	mmclienttypes "github.com/skip-mev/slinky/service/clients/marketmap/types"
 	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
 )
 
+const (
+	dYdXChain = "dydx"
+)
+
 var (
 	// oracleCfgPath is the path to write the oracle config file to. By default, this
 	// will write the oracle config file to the local directory.
 	oracleCfgPath = flag.String(
 		"oracle-config-path",
 		"oracle.json",
-		"path to write the oracle config file to",
+		"path to write the oracle config file to. this file is required to run the oracle.",
 	)
 
 	// marketCfgPath is the path to write the market config file to. By default, this
 	// will write the market config file to the local directory.
 	marketCfgPath = flag.String(
 		"market-config-path",
 		"market.json",
-		"path to write the market config file to",
+		"path to write the market config file to. this file is required to run the oracle.",
 	)
 
-	// providersOpt defines an optional list of providers to include in the local
-	// market config file. If this is not provided, all providers will be included.
-	providersOpt = flag.String(
-		"providers",
+	// chain defines the chain that we expect the oracle to be running on. Currently only
+	// supprting dYdX.
+	chain = flag.String(
+		"chain",
 		"",
-		"optional list of providers to include in the local market config file",
+		"chain that we expect the oracle to be running on. ex dydx.",
 	)
 
-	// tickersOpt defines an optional list of tickers to include in the local market
-	// config file. If this is not provided, all tickers will be included.
-	tickersOpt = flag.String(
-		"tickers",
+	// nodeURL is the URL of the validator. This is required if running the oracle with a market map provider.
+	nodeURL = flag.String(
+		"node-http-url",
 		"",
-		"optional list of tickers to include in the local market config file",
+		"URL of the dYdX node. this is required if running the oracle on the dYdX chain. this is the http address of the dYdX node.",
```

Comment:
I think this string should not make any references to dYdX.  This is just the "node market map source"